lib/generator/print_helpers.dart

```diff
@@ -115,15 +115,17 @@ Future<GraphQLResponse<${definition.queryName}>> execute${definition.queryName}Q
     }
 
     buffer.writeln('''    }),
-    headers: {
-      'Content-type': 'application/json',
-      'Accept': 'application/json',
-    },
+    headers: (client != null)
+        ? null
+        : {
+            'Content-type': 'application/json',
+            'Accept': 'application/json',
+          },
   );
 
-  final jsonBody = json.decode(dataResponse.body);
+  final dynamic jsonBody = json.decode(dataResponse.body);
```

Comment:
I think you can type it as `Map<String, dynamic>`  to be a little more strict

Reply:
Cool. I just went with the recommendations given by the editor :)